homeassistant/components/reolink/sensor.py

```diff
@@ -21,14 +21,30 @@
 
 from . import ReolinkData
 from .const import DOMAIN
-from .entity import ReolinkHostCoordinatorEntity
+from .entity import ReolinkChannelCoordinatorEntity, ReolinkHostCoordinatorEntity
+
+
+@dataclass
+class ReolinkSensorEntityDescriptionMixin:
+    """Mixin values for Reolink  sensor entities for a camera channel."""
+
+    value: Callable[[Host, int], float]
+
+
+@dataclass
+class ReolinkSensorEntityDescription(
+    SensorEntityDescription, ReolinkSensorEntityDescriptionMixin
+):
+    """A class that describes sensor entities for a camera channel."""
+
+    supported: Callable[[Host, int], bool] = lambda api, ch: True
 
 
 @dataclass
 class ReolinkHostSensorEntityDescriptionMixin:
     """Mixin values for Reolink host sensor entities."""
 
-    value: Callable[[Host], bool]
+    value: Callable[[Host], float]
```

Comment:
I think this is not related to this pr but a correction but is the wifi-signal a float?

Reply:
Correct, it is not related but I just noted it, no actually the wifi-singnal is a int, should I make it float | int?
Or just int?

I thought int would be a subset of float, but now that I think of it that is stupid in code (in mathematics it makes sence).

Reply:
If it's an int it's returning it should be an int...
I guess there is no typing here coming through as there is no complaints.

Reply:
I changed it to int, thanks!